wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -98,7 +98,7 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
         initViewModel()
 
         if (savedInstanceState == null) {
-            checkAlerts()
+            checkAlerts() //FIXME allet_test E/WindowManager: android.view.WindowLeaked: Activity de.schildbach.wallet.ui.MainActivity has leaked window DecorView@be910c7[] that was originally added here
```

Comment:
On a device with low storage space, one of the alerts will cause the app to crash.

Reply:
My comment above is not a request for a change, but an observation.

Reply:
Cool! This explained the issue. I'll fix it in this PR

Reply:
I managed to fix the crash that was occurring on my device, but am not sure if it will fix the one you described.